git_wrapper/tag.py

```diff
@@ -72,3 +72,15 @@ def push(self, name, remote, dry_run=False):
                 self.git_repo.git.push(remote, name)
         except git.GitCommandError as ex:
             raise_from(exceptions.PushException(msg), ex)
+
+    def list_tags(self):
+        """List git tags in the repository.
+        """
```

Comment:
(Nit) For documentation strings that fit on a single line, the pep8 recommendation is to place them on the same line (cf. https://www.python.org/dev/peps/pep-0008/#documentation-strings )